### PR TITLE
Api provides user to create Grocery Ping

### PR DESCRIPTION
**Pull request by pgaunitz**
### User Story
```
As a resident
In order to make a grocery run
I would like to be able to ping my community that I am going shopping
```
## Changes proposed in this pull request:
* Adds ping controller with create method
* Adds ping model
* Adds ping route
## What I have learned working on this feature:
* Repetition is.....

### CODE REVIEW

File: app/controllers/pings_controller.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+class PingsController < ApplicationController
+  def create
+    ping = Ping.create(ping_params)
+    if ping.persisted?
+      render json: { message: 'Your new trip is now active'}
```
Comment (kaylawoodbury) [inline app/controllers/pings_controller.rb]: Can we fix the formatting to have a space at the end of the curly to match the begining

File: spec/models/ping_spec.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe Ping, type: :model do
+  it 'should have valid Factory' do
+    expect(create(:ping)).to be_valid
+  end
+
+  describe 'Database table' do
+    it { is_expected.to have_db_column :time}
```
Comment (kfrostare) [inline spec/models/ping_spec.rb]: some curly's have space after opening curly but not before closing curly

File: app/controllers/pings_controller.rb (hunk 1 above)
Comment (kfrostare) [inline app/controllers/pings_controller.rb]: some curly's have space after opening curly but not before closing curly

File: app/controllers/pings_controller.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    else
+      render json: { message: 'You need to set a time for your new trip'},
+      status: 422
+    end
+
```
Comment (kaylawoodbury) [inline app/controllers/pings_controller.rb]: extra space

File: config/routes.rb [hunk 4]
```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
+  get 'pings/create'
```
Comment (kaylawoodbury) [inline config/routes.rb]: remove this

**Review by johanbounce — APPROVED** [review]

**Comment by johanbounce** [comment]
Cool pr!

**Review by kaylawoodbury — COMMENTED** [review]
left a couple of comments, otherwise this looks good to me!

**Comment by kfrostare** [comment]
Looks good to me now! /Kf

**Pull request merged**